lib/src/test/java/grpcbridge/common/TestSnakeService.java

```diff
@@ -0,0 +1,18 @@
+package grpcbridge.common;
+
+import grpcbridge.test.proto.SnakeTestServiceGrpc;
+import grpcbridge.test.proto.TestSnakeCase;
+import io.grpc.stub.StreamObserver;
+
+/**
+ * Test Implementation for TestSnakeService
+ */
+public class TestSnakeService extends SnakeTestServiceGrpc.SnakeTestServiceImplBase {
+
+    @Override
+    public void snake(TestSnakeCase.SnakeRequest request, StreamObserver<TestSnakeCase.SnakeResponse> responseObserver) {
```

Comment:
Long line. Can also import all the classes directly here and in other files, instead of using TestSnakeCase.SnakeResponse use SnakeResponse.